Submitted By: Jim Gifford <jim at linuxfromscratch dot org>
Date: 2006-07-04
Initial Package Version: 8.15.2
Origin: Various
Description: Make ijs-config Multilib Friendly

diff -Naur espgs-8.15.2.orig/ijs/ijs-config.in espgs-8.15.2/ijs/ijs-config.in
--- espgs-8.15.2.orig/ijs/ijs-config.in	2005-09-23 12:40:59.000000000 -0700
+++ espgs-8.15.2/ijs/ijs-config.in	2006-07-04 12:16:28.262157209 -0700
@@ -1,7 +1,6 @@
 #!/bin/sh
 
 prefix=@prefix@
-libdir=@libdir@
 exec_prefix=@exec_prefix@
 exec_prefix_set=no
 
@@ -44,11 +43,7 @@
       echo $includes
       ;;
     --libs)
-      case "x$libdir" in
-        x/lib | x/lib64 | x/usr/lib | x/usr/lib64) ;;
-        *) libdirs="-L$libdir" ;;
-      esac
-      echo $libdirs -lijs
+      echo -lijs
       ;;
     *)
       echo "${usage}" 1>&2
